aws/src/main/java/org/frankframework/filesystem/AmazonS3FileSystem.java

```diff
@@ -160,145 +165,166 @@ public void close() throws FileSystemException {
 	 */
 	@Override
 	public S3Object toFile(@Nullable String filename) {
-		S3Object object = new S3Object();
+		if (filename == null) {
+			return null;
+		}
+		S3Object.Builder builder = S3Object.builder();
 		int separatorPos = filename.indexOf(BUCKET_OBJECT_SEPARATOR);
-		if (separatorPos<0) {
-			object.setBucketName(bucketName);
-			object.setKey(filename);
+		if (separatorPos < 0) {
+			builder.key(filename);
 		} else {
-			object.setBucketName(filename.substring(0,separatorPos));
```

Comment:
Kan dit niet meer?

Reply:
Nee, dat zit niet in de builder en ook niet in het S3Object

Reply:
Gefixed in #7188.